### DISCUSSION TITLE
Edge to Stable?

### DISCUSSION THREAD

**Discussion by Botched1** [hassio-addons/addon-zwavejs2mqtt#667] [Q&A] [answered]
If this has been answered elsewhere, my apologies in advance! I searched, but didn't see this asked.

I see Edge was updated a few days ago. It makes sense that there is some lag between Edge and Stable, too. Not complaining about that.

- **I am curious, though, when does Edge typically move to Stable (assuming no issues found in Edge testing of course)?**

I couldn't tell looking at prior releases as it looked (unless I was reading it incorrectly) that for the last zwave js ui release Edge and Stable have the same release date (4/30).

**Comment by frenck** [comment, marked as answer]
> I am curious, though, when does Edge typically move to Stable (assuming no issues found in Edge testing of course)?


When I had the time to test and run the add-on on different machines that represent all supported installation methods and CPU architectures.

../Frenck

**Comment by Botched1** [comment]
That makes sense! Thanks for all you do!
